src/Utils/hiopOptions.cpp

```diff
@@ -1204,6 +1204,26 @@ void hiopOptionsNLP::ensure_consistence()
     }
   }
 
+  // use inertia-free approach if 1) solver is strumpack or cusolver-lu, or 2) if linsys is full
+  if(GetString("KKTLinsys")=="full") {
+    if(GetString("fact_acceptor")=="inertia_correction") {
+      if(is_user_defined("fact_acceptor")) {
+        log_printf(hovWarning,
+                   "option fact_acceptor=inertia_correction was changed to 'inertia_free' since "
+                   "inertia correction doesen't support unsymmetric full KKT linear system.\n");
+      }
+      set_val("fact_acceptor", "inertia_free");
+    }
+  } else if(GetString("linear_solver_sparse") == "strumpack" || GetString("linear_solver_sparse") == "cusolver-lu") {
+    if(GetString("fact_acceptor")=="inertia_correction") {
+      if(is_user_defined("fact_acceptor") && is_user_defined("linear_solver_sparse") ) {
+        log_printf(hovWarning,
+                   "option fact_acceptor=inertia_correction was changed to 'inertia_free' since "
+                   "the given linear solver cannot provide inertia.\n");
```

Comment:
"Option fact_acceptor=inertia_correction was changed to 'inertia_free' since the requested linear solver '%s' does not support inertia computation.\n", GetString("linear_solver_sparse").c_str()